client/src/components/pages/TeacherProfile/index.css

```diff
@@ -0,0 +1,57 @@
+.teach-profile {
+  text-align: center;
+  padding: 40px 2px;
+}
+
+.box-img > img {
+  width: 170px;
+  height: 170px;
+  border-radius: 50%;
+}
+
+.teach-profile__top > .teacher-name {
+  margin: 10px 0;
+  color: #5a5858;
+}
+
+.teach-profile__top > .teacher-subject {
```

Comment:
both `.teacher-subject` and `.teacher-name` uniquely identify the element, no need to specify `.teach-profile__top`